app/manifest/v3/chrome.json

```diff
@@ -6,19 +6,5 @@
     "matches": ["https://metamask.io/*"],
     "ids": ["*"]
   },
-  "minimum_chrome_version": "80",
-  "permissions": [
-    "storage",
-    "unlimitedStorage",
-    "clipboardWrite",
-    "http://localhost:8545/",
```

Comment:
It looks like some of the permissions have been removed, do we no longer need them?

Reply:
Thanks for the explanation, so does that mean we consume the v2 base for the v3 files?

Reply:
I don't think so. I think v3 just has had a different set of permissions from v2, notably, we don't seem to need to add URLs to the permissions file.

For a more detailed view of the original state of this file, the changes that caused issues see here:

https://github.com/MetaMask/metamask-extension/pull/18218/files

This PR has now been reverted atomically here https://github.com/MetaMask/metamask-extension/pull/18336/files

So I'll remove the changes from this PR

Reply:
> we don't seem to need to add URLs to the permissions file.

Thanks that clears up the confusion for me.